system/jhtree/jhtree.ipp

```diff
@@ -114,6 +114,8 @@ protected:
     void init(KeyHdr &hdr, bool isTLK);
     void loadBloomFilters();
     const CJHSearchNode *getRootNode() const;
+
+    inline bool isTLK() const { return (keyHdr->getKeyType() & HTREE_TOPLEVEL_KEY) != 0; }
```

Comment:
I assume this is done this way so that the call in getNode is non-virtual. A comment to that effect might stop a future code tidying effort from reversing this change.

Reply:
I'll add a comment.